### PR TITLE
Release 0.22.0-rc.0

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

**Review by SuperQ — APPROVED** [review]
LGTM

**Review by w0rm — APPROVED** [review]
Exciting release! So many new features! 🎉

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+* [FEATURE] Add time-based muting to routing tree. #2393
+* [FEATURE] Support TLS and basic authentication on the web server. #2446
+* [FEATURE] Add OAuth 2.0 client support in HTTP client. #2560
+* [ENHANCEMENT] Add composite durations in the configuration (e.g. 2h20m). #2353
+* [ENHANCEMENT] Add follow_redirect option to disable following redirects. #2551
+* [ENHANCEMENT] Add metric for permanently failed notifications. #2383
+* [ENHANCEMENT] Add support for custom authorization scheme. #2499
+* [ENHANCEMENT] Add support for not following HTTP redirects. #2499
+* [ENHANCEMENT] Add support to set the Slack URL from a file. #2534
+* [ENHANCEMENT] amtool: Add alert status for extended and simple output. #2324
```
Comment (beorn7) [inline CHANGELOG.md:17]: I had difficulty understanding what this is about. Perhaps `for` → `to` would make it clearer?

**Review by beorn7 — APPROVED** [review]
It has been almost a year. No wonder there are many changes…

**Review by simonpasquier — APPROVED** [review]
lgtm :bow: for doing this

**Pull request merged**